.ci/jenkins/jenkins-trigger.groovy

```diff
@@ -0,0 +1,44 @@
+// This script acts as a trigger script for the main ngraph-unittest.groovy
+// Jenkins job.  This script is part of a Jenkins multi-branch pipeline job
+// which can trigger GitHub jobs more effectively than the GitHub Pull
+// Request Builder (GHPRB) plugin, in our environment.
+
+// The original ngraph-unittest job required the following parameters.  We
+// set these up below as global variables, so we do not need to rewrite the
+// original script -- we only need to provide this new trigger hook.
+//
+// Parameters which ngraph-unittest uses:
+String  PR_URL = CHANGE_URL
+String  PR_COMMIT_AUTHOR = CHANGE_AUTHOR
+String  JENKINS_BRANCH = "chrisl/new-ci-trigger"
+Integer TIMEOUTTIME = "3600"
+// BRANCH parameter is no loner needed
+// TRIGGER_URL parameter is no longer needed
+
+// Constants
+JENKINS_DIR="."
+
+env.MB_PIPELINE_CHECKOUT = true
+
+node("bdw && nogpu") {
+
+    deleteDir()  // Clear the workspace before starting
+
+    // Clone the cje-algo directory which contains our Jenkins groovy scripts
+    git(branch: JENKINS_BRANCH, changelog: false, poll: false,
```

Comment:
I agree that `poll` does not belong here, but `changelog` might. This would render the list of changes/commits from the jenkins repo on the build page in addition to the ones from this repo. Not sure if that would be valuable for folks to see...

Reply:
Let's leave "changelog: false".  In a perfect world, only the git revision for ngraph would be displayed on the job page, because there is no labelling on what git revision is for what repo and having three unlabelled git SHAs is too confusing.  That said, even with "changelog: false", the git revision for cje-algo *does* appear on the job page (unfortunately).